strongbox-data-service/src/main/java/org/carlspring/strongbox/config/ConnectionConfigOrientDB.java

```diff
@@ -15,7 +15,8 @@
     private static final Logger logger = LoggerFactory.getLogger(ConnectionConfigOrientDB.class);
 
     public static final String PROPERTY_PROFILE = "strongbox.orientdb.profile";
-
+    public static final String PROPERTY_STUDIO_ENABLED = "strongbox.orientdb.studio.enabled";
```

Comment:
@carlspring you can disable Studio with `-Dstrongbox.orientdb.studio.enabled=false`. By default it's enabled.

Reply:
Actually, I think it would probably be better to have this disabled by default. The reason is this -- this is not an application that we, ourselves, are developing. If it has vulnerabilities, we'll most-likely  find out about them a bit late. We wouldn't like to expose our users to an unnecessary threat. This functionality would be useful for the following things:
- Development
-  Debugging
- Analyzing issues

@fuss86 , @steve-todorov : What do you guys think?

We could have this enabled in our projects during development (in the `strongbox-web-core` and `strongbox-distribution`).

Reply:
@carlspring we can have it explitely disabled in Jetty distribution, but it will be enabled by default for devalopment.

Reply:
Yes, these are valid points. Maybe we should have this enabled only on demand whenever a specific property is passed.

Reply:
@carlspring @steve-todorov regarding our discussion in the channel, I confirm that in the future we can make "proxy" controller which will pass Spring managed requests into Studio.

Reply:
@sbespalov ,

Okay! Would you mind raising a follow-up task for this, please?

Reply:
yes, will do it tomorrow